website/docs/reference/dbt-jinja-functions/dbt_version.md

```diff
@@ -8,7 +8,12 @@ description: "Read this guide to understand the dbt_version Jinja function in db
 The `dbt_version` variable returns the installed version of dbt that is
 currently running. It can be used for debugging or auditing purposes.
 
-### Example Usage
+## Versioning
```

Comment:
I like the idea of keeping the detailed information all on one page. This page in particular feels like a odd one to link to from elsewhere, since it's specifically about a dbt-Jinja context variable.

What do you think about moving this section to https://docs.getdbt.com/reference/commands/version? And in that page, mention how:
- For the dbt Core CLI, `dbt --version` will return the version of the installed version of `dbt-core` (plus adapters), which is all one big local bundle
- For the dbt Cloud CLI, `dbt --version` (CLI command) will return the version of the installed dbt Cloud CLI, and other `dbt_version` values will be the version of the "latest" dbt runtime in Cloud

Reply:
sure thing, np! 

the ticket specifically listed https://docs.getdbt.com/reference/dbt-jinja-functions/dbt_version. sorry for the misunderstanding!